pkg/dwarf/line/state_machine.go

```diff
@@ -189,8 +189,7 @@ func (lineInfo *DebugLineInfo) AllPCsBetween(begin, end uint64, excludeFile stri
 // copy returns a copy of this state machine, running the returned state
 // machine will not affect sm.
 func (sm *StateMachine) copy() *StateMachine {
-	var r StateMachine
-	r = *sm
+	r := *sm
```

Comment:
TBH I prefer this (and the similar one in pkg/proc/winutil) the way it is because it's making a copy of a struct, which is an unusual operation. It seems easier to understand to me, with a separate declaration.